### DISCUSSION TITLE
rgb_array as output

### DISCUSSION THREAD

**Discussion by svaichu** [empriselab/RCareWorld#91] [Help - Programming] [answered]
Hi!,

Is there any way I can get rgb_array as output in headless mode, so that I can export as a video using a gymnasium wrapper?

Thank you.

**Comment by YoruCathy** [comment, marked as answer]
Hi @svaichu , you can try these commands:
On your ubuntu server, do

```
sudo apt-get install xserver-xorg libglu1-mesa-dev freeglut3-dev mesa-common-dev libxmu-dev libxi-dev
```
Then, restart your server and connect your server to a screen. The screen won't render anything, but there must be a display device connecting to your server.

Then use the following commands
```
sudo nvidia-xconfig -a --use-display-device=None --virtual=1280x1024

export DISPLAY=:1

sudo /usr/bin/X :1 &
```

In your Python code, set `graphics=False`. Then you should be able to get the image in headless mode.
